x-pack/solutions/security/plugins/security_solution/public/management/components/console/service/parsed_command_input.ts

```diff
@@ -25,6 +28,9 @@ const parseInputString = (rawInput: string): ParsedCommandInput => {
       ? []
       : input.substring(inputFirstSpacePosition).trim().split(/--/);
 
+  // Find command definition for selector argument handling
+  const commandDef = commandDefinitions.find((def) => def.name === response.name);
```

Comment:
Same for changes in this module. its purpose is to only focus on parsing the text the user entered (this is the input parser I often refer to)... It should not be responsible for associating that text entered/parsed to a known command/argument.

I think that most of these changes should also be moved to `x-pack/solutions/security/plugins/security_solution/public/management/components/console/components/command_input/command_input.tsx:152` 

Keep the separation of concerns will make it easier to switch out this parser with something else in the future.